build.gradle.kts

```diff
@@ -63,7 +63,6 @@ tasks {
 
   named<GithubReleaseTask>("githubRelease") {
     dependsOn(downloadArtifacts)
-    setToken(project.properties["github.token"]?.toString() ?: "")
```

Comment:
Is there a reason you removed the setting of the token? I'm assuming this shouldn't affect your builds as you shouldn't be attempting a GitHub release.

Reply:
Sure, IIUC the method was removed from the plugin, which will fetch the `github.token` property by default, so the net effect should be no change on your end.

Of course I could not really test this.